packages/yswaps/scripts/libraries/dexes/solidly.ts

```diff
@@ -21,6 +21,7 @@ export type SwapResponse = {
 };
 
 export const getBestPathEncoded = async (swapParams: SwapParams): Promise<SwapResponse> => {
+  const { tokenIn, tokenOut, hopTokensToTest, amountIn } = swapParams;
   swapParams.hopTokensToTest = swapParams.hopTokensToTest ?? [];
```

Comment:
```suggestion
  const { tokenIn, tokenOut, amountIn } = swapParams;
```
Not going to be used.